### DISCUSSION TITLE
Czech translation

### DISCUSSION THREAD

**Discussion by AloisSeckar** [vuejs-translations/guidelines#64] [General]
I started with translation into Czech language. Although developers here in Czechia generally can speak sufficent English to go through the original docs as well, I think it would help make Vue.js easier accessible for young people who are looking around to start their developer carriers. And I want to see my country as a capital of IT in the heart of Europe one day, so I want to help it happen.

If you want to help me in this case, I'd be happy to welcome you on board. I do some lesser team leading in my job already, so hopefuly we can manage it here as well :)

Repo: https://github.com/AloisSeckar/vuejs-docs-cs
Site: https://vuejs-docs-cs.netlify.app/

**Comment by AloisSeckar** [comment]
I have couple of questions about things, I was unable to locate in the codebase yet.

Providing it is possible somehow, where can I translate:
- "Released under the MIT licence." on landing page?
- "ON THIS PAGE" header in the right column of most pages?
- "Search" placeholder in Algolia search input?
- 404 "Page not found" error page?

Also it turns out we would need another font, because current doesn't support well some special characters of our alphabet very well and it looks weird. Is it possible to set the font-family? Where is the best place to do it? 

![image](https://user-images.githubusercontent.com/7399922/218873677-2a5a9fbc-4e5d-4055-abf5-cc466660946d.png)

**Comment by AloisSeckar** [comment]
Status update:
Guide - Essentials and most of the stuff around (Ecosystem, Styling guide, etc) is translated (first try, without a proofread)
Moving onto Tutorial now, then back to advanced guide topics

**Comment by AloisSeckar** [comment]
Hello after a while. I would like to announce, that **I have completed the initial :czech_republic: translation for all docs pages**, including the most recent 3.4 update. :tada: 

Surely there will be more work to do with seeking missed parts, checking for typos and polishing the actual translation, but I believe it now qualifies for _"more than 50% of content"_. :blush:

Therefore I would like to ask for help with moving the translation from my personal fork to dedicated repository inside this organizazion.

Now I just want to clarify what I need to do:

- I was keeping [my fork](https://github.com/AloisSeckar/vuejs-docs-cs) in sync with the upstream on `main` branch
- I created [`translation`](https://github.com/AloisSeckar/vuejs-docs-cs/tree/translation) branch for the actual translation work

I recon I should merge `translation` into `main` and **then** go through detach process with GitHub AI assistant?

**Reply by Jinjiang** [reply]
@AloisSeckar nice job! To proceed with the guidelines, now you can try to detach your fork. Then we can transfer the repo into vuejs-translations org and have an official deployment further. I will also invite you into the org later. Cheers!

**Reply by AloisSeckar** [reply]
Thank you! Now I have one [open PR](https://github.com/vuejs/docs/pull/2641) from the fork to vuejs docs main, so I'd wait when it is merged/closed, do one more sync and then proceed with the detach asap.

**Reply by AloisSeckar** [reply]
[AloisSeckar/docs-cs](https://github.com/AloisSeckar/docs-cs) is now detatched

**Reply by Jinjiang** [reply]
Cool. Then you can transfer the repo to vuejs-translations org on your own.

**Reply by AloisSeckar** [reply]
It is done

**Reply by Jinjiang** [reply]
Well deployed now. Congrats 
https://cs.vuejs.org/

**Reply by Jinjiang** [reply]
Now you can add a link to the main doc site like https://github.com/vuejs/docs/pull/2579 or https://github.com/vuejs/docs/pull/2508

**Reply by AloisSeckar** [reply]
PR ready https://github.com/vuejs/docs/pull/2663

**Comment by AloisSeckar** [comment]
I opened couple of [questions in docs-cs repo](https://github.com/vuejs-translations/docs-cs/issues?q=is%3Aissue+is%3Aopen+label%3Aquestion) I would some guidance and help. I wanted to ask - what is the best way to draw attention to those and possible others in the future?

Should I:
- tag someone from the maintainers directly in each issue?
- open a new discussion here (or somewhere else)?
- reach someone from the team another way (email, chat)?
- do something else?

**Reply by Jinjiang** [reply]
A new discussion is welcome since the topic could be for not only docs-cs. Also we have a Discord channel to chat
https://github.com/orgs/vuejs-translations/discussions/77

**Comment by AloisSeckar** [comment]
I went through all pages once again to fix typos and polish the translation if necessary. I believe now everything is ready to make the translation official. PR here: https://github.com/vuejs/docs/pull/2772
